Corona-Warn-App/src/main/res/layout/trace_location_organizer_category_fragment.xml

```diff
@@ -19,6 +20,7 @@
         android:layout_width="match_parent"
         android:layout_height="0dp"
         android:layout_weight="1"
+        tools:listitem="@layout/trace_location_organizer_category_item"
         app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager" />
```

Comment:
Please reformat? I think `tools:` is at the very bottom

Reply:
done